### PR TITLE
Docstrings pep8

### PR DESCRIPTION
**Pull request by SRich-ing**
Completed docstrings
Brought code in line with PEP-8 standards

### CODE REVIEW

**Comment by mariatzele** [comment]
Why did you open another Pull Request for this? You already had https://github.com/mariatzele/university-app/pull/9 . Are you cancelling that?

File: src/main.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
+"""
+main.py
```
Comment (mariatzele) [inline src/main.py:2]: Why are the file names in top comments? This is not PEP8 standard

File: src/gui/components/listview.py [hunk 2]
```diff
@@ -15,8 +19,11 @@ def __init__(self, primary, checked_boxes, record_data):
         self.update_data()
 
     def create_listview(self):
+        """
+        Create and configure the list view widget based on checked columns.
+        """
         columns = [column for column, is_checked in self.checked_boxes if is_checked]
-        treeview = ttk.Treeview(self.primary, show="headings")
+        # treeview = ttk.Treeview(self.primary, show="headings")
```
Comment (mariatzele) [inline src/gui/components/listview.py]: I do not understand why this was commented out, won't this cause the headings to be missing?

File: src/gui/components/listview.py [hunk 3]
```diff
@@ -1,8 +1,12 @@
+"""listview.py"""
```
Comment (mariatzele) [inline src/gui/components/listview.py]: File name comments at the top of the file are not part of PEP8

**Review by mariatzele — CHANGES_REQUESTED** [review]
This cannot be merged as is. Review the PEP8 Standard please:
https://peps.python.org/pep-0008/

The filenames as top comments is not mentioned there so I don't know why they are added. More importantly a treeview component is commented out in `listview.py` meaning it won't be shown. Based on this code and pull request details I don't see why this was done

**Comment by SRich-ing** [comment]
> Why did you open another Pull Request for this? You already had #9 . Are you cancelling that?

I think deleting that additional .py file on the remote repository made it difficult for me to merge directly so it was easier to make a new branch in the time available. The earlier one can be merged or deleted, they don't conflict.

**Comment by SRich-ing** [comment]
> This cannot be merged as is. Review the PEP8 Standard please: https://peps.python.org/pep-0008/
> 
> The filenames as top comments is not mentioned there so I don't know why they are added. More importantly a treeview component is commented out in `listview.py` meaning it won't be shown. Based on this code and pull request details I don't see why this was done

The modules require a header according to pylint - I updated with descriptions. 

The treeview component should have been deleted. It doesn't do anything anymore.

File: src/gui/components/listview.py (hunk 2 above)
Comment (SRich-ing) [reply src/gui/components/listview.py]: it's a duplicate. See the next row

**Comment by SRich-ing** [comment]
> Completed docstrings Brought code in line with PEP-8 standards

Application has been tested with all five filters and tests run and confirmed to be OK

**Pull request merged**